.gitignore

```diff
@@ -5,3 +5,10 @@ __pycache__/
 .tox
 venv/
 src/xopen/_version.py
+settings.json
```

Comment:
Please undo these changes as they are unrelated.

IMO, a `.gitignore` file should only contain project-specific patterns. There’s also a global ignore file at `~/.config/git/ignore` where one can put things like `.vscode` and `.DS_Store` that are specific to the tools used on the machine.